Documentation/install/system_requirements.rst

```diff
@@ -145,5 +136,3 @@ and ``tc``, which is part of iproute2, to load BPF programs into the kernel.
 The minimum version of iproute2_ must be >= 4.8.0. Please see
```

Comment:
Does this also need to track the kernel version? probably not

Reply:
Not sure If I get this one.

Reply:
It was the exact version as the minimum kernel. I'm not sure whether they are tightly coupled or not (but I doubt it). So I'm asking (@borkmann) if we need to up this number to 4.9.2 as well.

Reply:
No tight coupling, iproute >= 4.8.0 is good.